doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -211,6 +211,19 @@ nRF9160 samples
     * Support for injecting GNSS reference altitude for low accuracy mode.
       For a position fix using only three satellites, GNSS module must have a reference altitude that can now be injected using the ``gnss agps ref_altitude`` command.
 
+* :ref:`modem_trace_backend_sample` sample:
```

Comment:
merge issue?

Reply:
Weird. Not sure how that ended up in here. Thanks for catching that.